changelog/unreleased/issue-1433

```diff
@@ -0,0 +1,10 @@
+Enhancement: Support UTF-16 encoding and process Byte Order Mark
+
+On Windows, text editors commonly leave a Byte Order Mark at the beginning of
+the file to define which encoding is used (oftentimes UTF-16). We've added code
+to support processing the BOMs in text files, like the exclude files, the
+password file and the file passed via `--files-from`.
```

Comment:
I'd explicitly mention that text file encoding detection is NOT used for files being backed up.